package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "mongodb": "^5.5.0",
+    "mongoose": "^7.2.1"
+  }
+}
```

Comment:
might have accidentally install the dependencies in root. should be fine though.

Reply:
Oh yea oops I only removed the node module